spec.html

```diff
@@ -39904,6 +39904,57 @@ <h1 id="ao-issuperreference">IsSuperReference ( _V_ )</h1>
         1. Return ? ForIn/OfBodyEvaluation(|BindingIdentifier|, |Statement|, _keyResult_, ~enumerate~, ~varBinding~, _labelSet_).
       </emu-alg>
     </emu-annex>
+
+    <emu-annex id="sec-html-document-dot-all-exotic-object">
+      <h1>HTML Document Dot All Exotic Object</h1>
+      <p>An <dfn>HTML document dot all exotic object</dfn> is an exotic object that behaves like undefined in the <emu-xref href="#sec-toboolean">ToBoolean</emu-xref> and <emu-xref href="#sec-abstract-equality-comparison">Abstract Equality Comparison</emu-xref> abstract operations and when used as an operand for the <emu-xref href="#sec-typeof-operator">typeof operator</emu-xref>. HTML document dot all exotic objects have all the ordinary object internal methods and slots.</p>
+      <emu-note>
+        <p>These objects are never created by this specification. However, the <a href="https://html.spec.whatwg.org/multipage/obsolete.html#dom-document-all"><code>document.all</code> object</a> in web browsers is a host-created HTML document dot all exotic object that exists for web compatibility purposes. There are no other known examples of this type of object.</p>
```

Comment:
This was discussed somewhat, I think there was resistance to "overreach", but I'm definitely fine with the note saying something to this effect.